anyway/flask_app.py

```diff
@@ -1579,6 +1613,67 @@ def get_role_object(role_name):
     return role
 
 
+@app.route("/admin/update_user", methods=["POST"])
+def admin_update_user() -> Response:
+    allowed_fields = [
+        "user_current_email",
+        "first_name",
+        "last_name",
+        "email",
+        "phone",
+        "user_type",
+        "user_url",
+        "user_desc",
+        "is_user_completed_registration",
+    ]
+
+    res = check_admin_api_with_input(request, allowed_fields)
```

Comment:
Usually a function returns `None` if it failed, or did not find anything. I prefer to rename the function to reflect it, e.g. `look_for_errors_in_request`(). Alternatively, it can return `Tuple{bool, Optional[Request]]`, were it sill return `(True, None)` for success, and `(False, Rsponse)` for failure.